pkg/workflows/workflows_test.go

```diff
@@ -25,24 +25,29 @@ func TestCreateWorkflows(t *testing.T) {
 	deployType := "helm"
 	flagVariables := []string{}
 	templatewriter := &writers.LocalFSWriter{}
-	flagValuesMap := map[string]string{"AZURECONTAINERREGISTRY": "testAcr", "CONTAINERNAME": "testContainer", "RESOURCEGROUP": "testRG", "CLUSTERNAME": "testCluster", "BRANCHNAME": "testBranch"}
+	flagValuesMap := map[string]string{"AZURECONTAINERREGISTRY": "testAcr", "CONTAINERNAME": "testContainer", "RESOURCEGROUP": "testRG", "CLUSTERNAME": "testCluster", "BRANCHNAME": "testBranch", "BUILDCONTEXTPATH": "."}
```

Comment:
looks like these are duplicates again?

Reply:
Yup. Missed it when copying over. Great catch!